autonomy/cli/deploy.py

```diff
@@ -194,39 +194,6 @@ def run(build_dir: Path, no_recreate: bool, remove_orphans: bool) -> None:
     run_deployment(build_dir, no_recreate, remove_orphans)
 
 
-def run_deployment(
```

Comment:
Was this a complete copy& paste?

Reply:
Yeah, I just moved it below.

Reply:
For the docs, the CLI docs does not need updating but we'll have to add a tutorial on publishing services, deploying them on-chain and running agent instances. I had a talk with jose about this, I'll create appropriate tickets for him.